packages/dialog/tests/DialogTitle.test.tsx

```diff
@@ -13,6 +13,7 @@ test('should render close button if prop set', async () => {
   const close = jest.fn();
   const { container } = render(<DialogTitle onClose={close}>Foobar</DialogTitle>);
 
+  // eslint-disable-next-line testing-library/no-container, testing-library/no-node-access
   const closeBtn = container.querySelector('button');
```

Comment:
I didn't refactor the cases where we use a lot of `container`. But cases like these should rather be
```
screen.getByRole("button")
```